subprojects/launcher/src/test/groovy/org/gradle/launcher/cli/BuildActionSerializerTest.groovy

```diff
@@ -50,6 +49,7 @@ class BuildActionSerializerTest extends SerializerSpec {
         result.startParameter.taskNames == ['a', 'b']
     }
 
+    @Unroll
```

Comment:
Why did you remove the `@Unroll` from the class?

Reply:
I assume because the other tests don't need to unroll. It seems to be the pattern we usually use (annotate only the methods that actually `Unroll`). Or should we do it differently? (I don't care much, I just like consistency :)).

Reply:
I assume because the other tests don't need to unroll. It seems to be the pattern we usually use (annotate only the methods that actually Unroll).

My experience with this is that the net result is lots of tests with missing @Unroll, which makes the diagnostics worse.

Reply:
+1 to what @ldaley said: When we don't put the `@Unroll` on the class most times we end up missing with an `@Unroll` on a test method. That is why I started moving the `@Unroll` on the class level when we change a test class. 

Is there any disadvantage for the `@Unroll` on the class level? There was a problem with IDEA navigation from the test to the source code, though that has been fixed for a very long time now.

Reply:
No disadvantange for `@Unroll` on test class level, just because only one or two test methods in this class have `where` block. We can put the `@Unroll` back of course.